### ISSUE TITLE
Error generating habit stats calendar window when user has enabled 'calendar-mark-holidays-flag'

### ISSUE THREAD

**Issue by wombatzus** [ml729/org-habit-stats#6]
Initial call to `calendar-generate-window` errors out when calendar attempts to mark holidays, as habit-stats attempts to create a calendar centered 1 month previous to current, but calendar tries to mark holidays based on current month and thus gets error attempting to move-column to a negative number.  Error looks like "Wrong type argument: wholenump, -19" 

I suggest let-binding `calendar-mark-holidays-flag` and `calendar-mark-diary-entries-flag` to nil around initial call to `calendar-generate-window` in `org-habit-stats-make-calendar-buffer` and `org-habit-stats-refresh-calendar-buffer`, that seems to work for me.

This is on emacs 30.0.50 built from source 2024-01-22.

**Comment by vberezhnev** [comment]
Hi, did you solve your problem?

**Comment by wombatzus** [comment]
Yes, I modified the code locally as I described above and it worked for me.
